### DISCUSSION TITLE
DivestOS Versions

### DISCUSSION THREAD

**Discussion by ghost** [Divested-Mobile/DivestOS-Build#363] [Q&A] [answered]
I looked in Device Downloads and saw 20.0 / 13.0 does that mean that this Devices are on DivestOS 20 with Android 13? I'm confused because I saw on DivestOS Github: 21.0 which I'm guessing means DivestOS 21.0 ? Let's discuss about my Device. It is the Xiaomi Poco X3 NFC (surya) and on the To-Do List it says 20.0 does that mean that my Device will get DivestOS 20 with Android 13 or will it get Android 14/15 and maybe DivestOS 21.0 ? Because my Device is supported by LineageOS with Android 14 and it will probably get Android 15 in the coming Months. So how does that make sense? I don't understand why it would get lower Android Version with DivestOS? Or am I wrong? And why is not DivestOS 21.0 listed in Device Downloads with the Devices which has DivestOS 21.0 support? (I'm just confused and want to learn)

**Comment by ghost** [comment]
If someone knows the Answer to this, I would appreciate, if it could be shared with me.

**Comment by steadfasterX** [comment, marked as answer]
19, 20.0, etc are **LineageOS** versions.
the version behind that is the actual **Android** version. 

divest 21 (which is again the LOS version and is itself based on Android 14) has not been released yet. the fact that you see things in the github repo(s) related to 21 are because it is planned to support 21 and thats all about it.
the only valid way to see what divest OS version your device supports is looking at the device downloads page.

**Reply by ghost** [reply]
Thank you so much for explaining it! Oh nice, is there an Info when Divest 21 will be out there? But I guess that there will be Divest 22 too, because of Android 15.
Yes, true.

**Reply by steadfasterX** [reply]
no info or ETA is communicated, just that it will come.

**Reply by ghost** [reply]
Okay, thank you! It is a bit far behind in terms of Android Version but hopefully it will come.
